.github/workflows/build_and_test.yml

```diff
@@ -306,6 +306,12 @@ jobs:
         name: unit-tests-log-${{ matrix.modules }}-${{ matrix.comment }}-${{ matrix.java }}-${{ matrix.hadoop }}-${{ matrix.hive }}
         path: "**/target/*.log"
 
+  maven-test:
```

Comment:
Let's make another attempt. This is to confirm that the GA can be successfully built. Once it can be successfully built, we will delete this piece of code.